app/views/users/edit.html.erb

```diff
@@ -0,0 +1,17 @@
+
+<h1>Update User: <%=@user.username %></h1>
+<%= form_for @user do |f| %>
```

Comment:
- [ ] [OPTIONAL] To avoid repeating the form twice you can use a partial and call it both on new and edit.

Reply:
Integrated ruby code has now been moved into a '_form.html.erb' partial and rendered in 'new' and 'update' views.